assets/memberships-gate/editor.js

```diff
@@ -63,6 +87,27 @@ const GateEdit = ( { editPost, createNotice, meta } ) => {
 						) }
 					/>
 				) }
+				{ meta.style === 'overlay' && (
+					<Fragment>
+						<SelectControl
+							label={ __( 'Size', 'newspack' ) }
+							value={ meta.overlay_size }
+							options={ overlaySizes }
+							onChange={ value => editPost( { meta: { overlay_size: value } } ) }
+						/>
+						<PositionControl
+							label={ __( 'Position', 'newspack' ) }
+							value={ meta.overlay_position }
```

Comment:
```suggestion
							value={ meta.overlay_position }
							size={ meta.overlay_size }
```

To enable the alternative `PositionControl` styles for full-width